controllers/login.js

```diff
@@ -280,6 +281,8 @@ async function getOauthSystems(req) {
 		.catch((err) => logger.error('error loading oauth system list', formatError(err)));
 }
 
+const isDBC = () => SC_THEME === 'default';
```

Comment:
DO NEVER EVER ACCESS THIS VARIABLE

Reply:
You mean to add new variable and access it instead of SC_THEME?

Reply:
ok did it